ui/legacy/src/components/cstudio-forms/data-sources/components.js

```diff
@@ -30,7 +30,8 @@
     this.allowEmbedded = false;
     this.enableSearch = false;
     this.enableBrowse = false;
-    this.baseRepoPath = '/site/components';
+    this.defaultBaseRepoPath = '/site/components';
+    this.baseRepoPath = null;
     this.baseBrowsePath = '/site/components';
```

Comment:
Does the base browse path work correctly?

Reply:
yes it does, it gets overridden later in the code (if set in content-type editor prop)